jenkins/image_building/build-image.sh

```diff
@@ -35,7 +35,10 @@ export DIB_DEV_USER_USERNAME="metal3ci"
 export DIB_DEV_USER_PWDLESS_SUDO="yes"
 export DIB_DEV_USER_AUTHORIZED_KEYS="${current_dir}/authorized_keys"
 
-if [[ "${IMAGE_OS}" == "ubuntu" ]]; then
+if [[ "${IMAGE_OS}" == "ubuntu" ]] && [[ "${IMAGE_TYPE}" == "node" ]]; then
+  export DIB_RELEASE=noble
+  numeric_release=24.04
+elif  [[ "${IMAGE_OS}" == "ubuntu" ]] && [[ "${IMAGE_TYPE}" == "ci" ]]; then
   export DIB_RELEASE=jammy
   numeric_release=22.04
```

Comment:
```suggestion
if [[ "${IMAGE_OS}" == "ubuntu" ]]; then 
  if [[ "${IMAGE_TYPE}" == "node" ]]; then
    export DIB_RELEASE=noble
    numeric_release=24.04
  elif [[ "${IMAGE_TYPE}" == "ci" ]]; then
    export DIB_RELEASE=jammy
    numeric_release=22.04
  fi
```
This is easier to read, maintain and eventually to remove as well.